Assets/Scenes/Columbia/main.gd

```diff
@@ -21,8 +22,11 @@ enum server_packets {
 	PLAYER_POSITION_PARAMETERS_UPDATE = 8,
 	ROD_POSITION_PARAMETERS_UPDATE = 10,
 	USER_LOGIN_ACK = 13,
+	CHAT = 15,
 }
 
+signal chat_message(message)
```

Comment:
what is this line for?

Reply:
[> what is this line for?](https://github.com/OpenLWR/OpenLWR/pull/11#discussion_r1632410316)

everything else is looking good, but you still haven't answered by question

Reply:
sorry for the late reply
that line defines the `chat_message` signal which is emitted at [line 1791](https://github.com/kcinnaJlol/OpenLWR/blob/3dcd20a4b503674b6c3bc1be48c4171c16775186/Assets/Scenes/Columbia/main.gd#L1791) and is connected to `_on_chat_message` by [Chat.gd, line 6](https://github.com/kcinnaJlol/OpenLWR/blob/3dcd20a4b503674b6c3bc1be48c4171c16775186/Assets/Scenes/Player/Chat.gd#L6)

Reply:
i see